File: Arrays/ArraySum.cpp
#include<bits/stdc++.h>
using namespace std;


int sum(int* arr, int size){
    int ans{};
    for(int i{};i<size;i++){
        ans += arr[i];
    }
    return ans;
}

int main(){
    int size;
    cin >> size;
    int* arr{new int[size]};
    for(int i{}; i<size; i++){
        cin >> arr[i];
    }
    cout << sum(arr,size) << endl;
}

File: Arrays/ArraySum/ArraySum.py
def sumOfArrayElements(arr):
    sum = 0
    for i in arr:
        sum += i
    return sum

if __name__ == "__main__":
    n = int(input())
    arr = [int(i) for i in input().split()]
    ans = sumOfArrayElements(arr)
    print(ans)
